functions/New-DbaAgentJob.ps1

```diff
@@ -379,16 +410,23 @@ Creates a job with the name "Job One" on multiple servers using the pipe line
                     # Make sure the target is set for the job
                     Write-Message -Message "Applying the target (local) to job $Job" -Level Verbose
                     $smoJob.ApplyToTargetServer("(local)")
```

Comment:
This is done to set a target for the job. Only in the case of multi server administration with the agent will this need to be changed.
It doesn't have anything to do with the connection to the sql instance for the creation of that job